akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/ExternalInteractions.scala

```diff
@@ -153,12 +148,8 @@ private[akka] trait SnapshotInteractions[C, E, S] {
   }
 
   /** Deletes the snapshots up to and including the `sequenceNr`. */
-  protected def internalDeleteSnapshots(toSequenceNr: Long): Unit = {
+  protected def internalDeleteSnapshots(fromSequenceNr: Long, toSequenceNr: Long): Unit = {
```

Comment:
I just realized that we have:
```scala
protected def internalDeleteSnapshots(fromSequenceNr: Long, toSequenceNr: Long): Unit
protected def internalDeleteEvents(toSequenceNr: Long, lastSequenceNr: Long): Unit
```

For consistency, WDYT:
```scala
protected def internalDeleteEvents(lastSequenceNr: Long, toSequenceNr: Long): Unit
```

Reply:
lastSequenceNr is typically >= toSequenceNr, and it's not a fromSequenceNr, so I'm not sure that will improve consistency

Reply:
I mean having each function declare the toSequenceNr as the second param for consistency.

Reply:
ok